src/identity/tezos/TezosLedgerConnector.ts

```diff
@@ -0,0 +1,110 @@
+import Transport from "@ledgerhq/hw-transport";
```

Comment:
How much of this file is being unit tested?

Reply:
i'm not immediately aware of a way to mock usb responses.